src/components/admin/previewComponent/reportsPreview/ReportsPreview.tsx

```
@@ -0,0 +1,44 @@
+  const { allReportsData } = useGetAllReports();
+
+  const previewList = allReportsData
+    ? allReportsData.length > 6
+      ? allReportsData.slice(0, 4)
+      : allReportsData
+    : [];
+
+  return (
+    <S.Container>
+      {previewList?.map((report) => (
+        <S.Wrapper>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)